less/main.less

```diff
@@ -3,7 +3,7 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 @bodyBG: #000;
-@windowContainerFG: #f00;
+@windowContainerFG: #000;
```

Comment:
main.less is removed on https://github.com/brave/browser-laptop/pull/10861, not being sure what those specifications are required for. See: https://github.com/brave/browser-laptop/pull/10861/files#diff-eb0db025dc73e282655968f53c9bfbdfR171. We could discuss it later.

Reply:
yes I saw that just after pushing. Should be removed here too.

Reply:
Confirmed main.less was removed here too 👍